presto-parser/src/main/java/com/facebook/presto/sql/parser/SqlParser.java

```diff
@@ -231,9 +232,10 @@ public void exitNonReserved(SqlBaseParser.NonReservedContext context)
             context.getParent().removeLastChild();
 
             Token token = (Token) context.getChild(0).getPayload();
-            if (token.getText().equalsIgnoreCase("CURRENT_ROLE")) {
-                warningConsumer.accept(new ParsingWarning(format("Reserved word used: %s", token.getText()), token.getLine(), token.getCharPositionInLine()));
+            if (RESERVED_WORDS_WARNING.contains(token.getText().toUpperCase())) {
+                warningConsumer.accept(new ParsingWarning(format("This will soon be made a reserved word: %s", token.getText()), token.getLine(), token.getCharPositionInLine()));
```

Comment:
`"%s should be a reserved word, please use double quote (\"%s\"). This will be made a reserved word in future release."`

Reply:
thanks!